stock_move_location/wizard/stock_move_location.py

```diff
@@ -222,11 +229,16 @@ def action_move_location(self):
         return self._get_picking_action(picking.id)
 
     def _get_picking_action(self, pickinig_id):
-        action = self.env.ref("stock.action_picking_tree_all").read()[0]
```

Comment:
This is correct, but should be done with sudo

Reply:
Do you think it should be preferred to recreating an action as done below ?

Reply:
Nope, in fact, the good way is:

action = self.env['ir.actions.act_window']._for_xml_id("stock.action_picking_tree_all")

Reply:
Thanks, just did, it works !